### PR TITLE
use 'tf.nn.sigmoid_cross_entropy_with_logits' method

### PR DESCRIPTION
**Pull request by jihobak**
First of all, Thanks for your awesome resources.

In previous lab, we've learned
`hypothesis = tf.sigmoid(tf.matmul(X, W) + b)`
`cost = -tf.reduce_mean(Y * tf.log(hypothesis) + (1 - Y) *
                       tf.log(1 - hypothesis))`

Instead, What about introducing  `'tf.nn.sigmoid_cross_entropy_with_logits'`  simple method?

### CODE REVIEW

File: lab-05-2-logistic_regression_diabetes.py [hunk 1]
```diff
@@ -17,11 +17,12 @@
...
 
 # Hypothesis using sigmoid: tf.div(1., 1. + tf.exp(tf.matmul(X, W)))
-hypothesis = tf.sigmoid(tf.matmul(X, W) + b)
+logits = tf.matmul(X, W) + b
+hypothesis = tf.sigmoid(logits)
 
 # cost/loss function
-cost = -tf.reduce_mean(Y * tf.log(hypothesis) + (1 - Y) *
-                       tf.log(1 - hypothesis))
+losses = tf.nn.sigmoid_cross_entropy_with_logits(None, labels=Y, logits=logits)
```
Comment (hunkim) [inline lab-05-2-logistic_regression_diabetes.py]: loss = tf.reduce_mean(tf.nn.sigmoid_cross_entropy_with_logits( labels=Y, logits=logits)) ?

Comment (hunkim) [inline lab-05-2-logistic_regression_diabetes.py]: This is logistic, so I think it's OK without Z`sigmoid_cross_entropy_with_logits`

**Comment by jihobak** [comment]
@hunkim Thanks for your review :) I close this PR
